src/exoscale/deps_modules.clj

```diff
@@ -21,16 +17,17 @@
    :input-deps-edn-file "deps.edn"
    :versions-edn-file ".deps-versions.edn"
    :versions-edn-keypath []
-   :modules-dir "modules"})
+   :modules-patterns ["modules/*/deps.edn"]})
```

Comment:
`:modules-deps-edn-paths` ?

Reply:
I went with `:modules-deps-edn-patterns` to make it explicit it's a pattern